lib/IconClear.js

```diff
@@ -0,0 +1,3 @@
+import IconClear from '../src/components/Icon/IconClear';
+
+export default IconClear;
```

Comment:
We should consider using [babel-plugin-transform-export-extensions](https://babeljs.io/docs/plugins/transform-export-extensions/), which would allow this syntax instead:

```js
export default from '../src/components/...'
```

Reply:
Disregard, as per [comment below](https://github.com/Mudano/ui-react/pull/47#discussion_r166582945)